src/lib/grammar.rs

```diff
@@ -401,7 +401,7 @@ mod test {
                  | expr '-' expr
                  | expr '*' expr
                  | expr '/' expr
-                 | %prec '-'  expr '*'
+                 | '-'  expr %prec '*'
                  | 'id' ;
         ").unwrap());
         assert_eq!(grm.prod_precs.len(), 7);
```

Comment:
lrtable inserts a "start" rule with a single production. If your starting rules is `S`, it'll add `S': S $` (where `$` is the end token). So `prod_precs[0]` is the precedence of that starting rule.